kolibri/plugins/coach/assets/src/views/common/QuestionDetailLearnerList.vue

```diff
@@ -24,13 +24,13 @@
               v-if="learner.noattempt"
               class="item svg-item"
               :style="{ fill: $themeTokens.annotation }"
-              icon="cancel"
+              icon="notStarted"
             />
             <KIcon
               v-else-if="!learner.correct"
               class="item svg-item"
               :style="{ fill: $themeTokens.incorrect }"
-              icon="cancel"
+              icon="incorrect"
```

Comment:
This feels like it should be the right thing to use here... not sure why `cancel` was used for both of these icons. `cancel` no longer exists and I _assume_ it's what turned into `incorrect` given the `$themeToken` that was used here.

- `notStarted` is used when the learner has no attempts
- `incorrect` is used when the user answered incorrectly (and presumably needs help)